dockers/docker-fpm-frr/frr/bgpd/templates/general/policies.conf.j2

```diff
@@ -59,6 +60,9 @@ route-map FROM_BGP_PEER_V6 permit 1
 route-map FROM_BGP_PEER_V6 permit 100
 !
 route-map TO_BGP_PEER_V6 permit 100
+  call CHECK_IDF_ISOLATION
+!
+route-map CHECK_IDF_ISOLATION permit 10
```

Comment:
Why we to define CHECK_IDF _ISOLATION with permit 10 here, see it is present in dockers/docker-fpm-frr/frr/bgpd/idf_isolate/idf_isolate.conf.j2

Reply:
this route-map with permit 10 is the default configuration that gets invokes by the "call". idf_isolate.conf.j2 gets rendered only when idf_isolation is executed.